File: telerik_ui_for_winforms_courseware_chm/Courseware/Appearance/CS/ElementsAndHTMLStyling/ElementsAndHTMLStyling/ShapedForm1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls.UI;
using Telerik.WinControls;

namespace AnimatedProperty
{
    public partial class ShapedForm1 : Telerik.WinControls.UI.ShapedForm
    {
        public ShapedForm1()
        {
            InitializeComponent();
        }

        private void ShapedForm1_Load(object sender, EventArgs e)
        {
            ThemeResolutionService.ApplicationThemeName = "Aqua";
            ThemeList list = ThemeResolutionService.GetAvailableThemes(radTitleBar1);

            lblTitle.Text =
              "<html>" +
              "<size=11><b><color=1,78,194><font=Papyrus>Blue Sky Travel Planning</b>" +
              "<br>" +
              "<size=10><color=72,173,229><font=Narkisim>Get where you're going";
        }
    }
}

File: telerik_ui_for_winforms_courseware_chm/Courseware/Appearance/VB/ElementsAndHTMLStyling/ElementsAndHTMLStyling/ShapedForm1.vb
Imports Telerik.WinControls

Public Class ShapedForm1

    Private Sub ShapedForm1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ThemeResolutionService.ApplicationThemeName = "Aqua"
        Dim list As ThemeList = ThemeResolutionService.GetAvailableThemes(radTitleBar1)

        lblTitle.Text = "<html>" + "<size=11><b><color=1,78,194><font=Papyrus>Blue Sky Travel Planning</b>" + "<br>" + "<size=10><color=72,173,229><font=Narkisim>Get where you're going"
    End Sub
End Class
